lib/scholar/metrics/distance.ex

```diff
@@ -407,4 +426,61 @@ defmodule Scholar.Metrics.Distance do
     res = Nx.select(one_zero?, 0.0, res)
     1.0 - Nx.select(both_zero?, 1.0, res)
   end
+
+  @doc """
+  Hamming distance.
+
+  N_unequal(x, y) / N_tot
```

Comment:
```suggestion
  $hamming(x ,y) = \frac{#unequal(x, y)}{#x}$
```

Reply:
The denominator is really the length of the axis you're summing over. If we wanted to allow no axis, `Nx.mean` would then use the total number of items so again the denominator would be $m \times n$. I'm not sure of a succinct way of expressing this

Reply:
I've edited the suggestion to mention the aggregation axes.

I think for non-weighted we can support many axes and extend the weighted version when weighted_mean supports them.

Reply:
Also, `#` here symbolizes the size of the set

Reply:
Ah, thanks I didn't realize that about `#`. I'll add in support for many axes in non-weighted

Reply:
@polvalente Do you think it makes sense to change the `axes` validator to allow for `int`s for a single axis? Then within the validator just wrap the int in a list and pass that to the function for use as `:axes` parameter? This seems more intuitive than requiring a list of one item (but then again, the term `axes` suggests an enumerable type)

Reply:
I'd rather always have lists, mainly because it's the way that the :axes parameter works in all of Nx so far

Reply:
makes sense, thanks